internal/pkg/uploader/upload/info.go

```diff
@@ -0,0 +1,50 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package upload
+
+import (
+	"time"
+
+	"github.com/elastic/fleet-server/v7/internal/pkg/model"
+)
+
+// the only valid values of upload status according to storage spec
+type Status string
```

Comment:
[Suggestion | Go convention]
```suggestion
// Status represents the only valid values of upload status according to storage spec
type Status string
```